libretroshare/src/services/p3idservice.cc

```diff
@@ -4336,7 +4336,7 @@ void p3IdService::generateDummy_OwnIds()
 
 	/* grab all the gpg ids... and make some ids */
 
-	RsPgpId ownId = mPgpUtils->getPGPOwnId();
+	mPgpUtils->getPGPOwnId();
```

Comment:
just remove the line if ownId is not used.

Reply:
OK